pkg/graph/schema.graphql

```diff
@@ -1640,7 +1704,9 @@ type Query {
   relatedSystemIntakes(id: UUID!): [SystemIntake!]!
   trbRequest(id: UUID!): TRBRequest!
   trbRequestCollection(archived: Boolean! = false): [TRBRequest!]!
+  trbRequestAttendees(id: UUID!): TRBRequestAttendeesPayload!
```

Comment:
I think we could probably replace this distinct query by having the `type TRBRequest` have an `attendees: [TRBRequestAttendee!]!` type on it.